src/components/day.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react';
+import Cell from './cell';
+
+function Day({ row, mode, onChange }) {
+    const [range, setRange] = useState({
+        start: null,
+        end: null
+    });
+    const [mouseStatus, setMouseStatus] = useState({
+        down: false
+    });
+    const [grid, setGrid] = useState(row);
+
+    function handleMouseClick(cellID) {
+        setGrid((prev) => {
```

Comment:
@hanakamer, do you have the prettier working on your side? On mine the files are not indented properly.

In VSCode Settings, you need to have "Format on Save" enabled.

![image](https://user-images.githubusercontent.com/14869087/145911483-987cfb8f-f06d-4475-a902-6f848ac456f9.png)

Reply:
on my side it is working and corrects the code before saving. but maybe the space/tab settings are different? because I realised that if I delete the space before 'start' property, it saves the code as in here.

Reply:
I pulled the latest version and now it's all okay except this line:

![image](https://user-images.githubusercontent.com/14869087/146086141-3840aefa-950d-431e-abf1-36dba976e73f.png)

In VSCode, on the bottom bar it says the space/tab configuration you have in place: 
![image](https://user-images.githubusercontent.com/14869087/146086209-92a47114-cd5f-428c-9ff0-a630fefe53d2.png)

Do you also have Space: 2 ? I don't think that's the reason though...